### ISSUE TITLE
{:ok, {:ok, user}} returned from Addict.Repository.create

### ISSUE THREAD

**Issue by mrluc** [trenpixster/addict#54]
At least for me, I'm getting this behavior; it causes the last stage in registration, sending user email, to explode with:

```
(UndefinedFunctionError) undefined function: :ok.username/1 (module :ok is not available)
        :ok.username({:ok, %Flashbang.User{__meta__: %Ecto.Schema.Metadata{source: {nil, "users"}, state: :loaded}, email: "az", hash: "$pbkdf2-sha512$60000$tv82nurn18WwchXaLHdxhg$qDPWQVlr/lg3WQznHCms84CL/Oek8VQ93whhznVkyt6Qm.rlQPB4YA2JjLdKk.Vo8LSheUibSrkKSJ1KjSwGtw", id: 3, inserted_at: #Ecto.DateTime<2015-08-14T22:27:01Z>, recovery_hash: nil, updated_at: #Ecto.DateTime<2015-08-14T22:27:01Z>, username: "z"}})
        lib/addict/email_gateway.ex:7: Addict.EmailGateway.send_welcome_email/2
        lib/addict/interactors/addict_manager_interactor.ex:162: Addict.ManagerInteractor.send_welcome_email/2
        lib/addict/controller.ex:95: Addict.Controller.register/2
        lib/addict/controller.ex:89: Addict.Controller.phoenix_controller_pipeline/2
        (flashbang) lib/phoenix/router.ex:265: Flashbang.Router.dispatch/2
        (flashbang) web/router.ex:1: Flashbang.Router.do_call/2
        (flashbang) lib/flashbang/endpoint.ex:1: Flashbang.Endpoint.phoenix_pipeline/1
```

After digging a bit I believe a recent commit that touched `Addict.Repository.create` caused this.

**Comment by trenpixster** [comment]
@mrluc what ecto and phoenix versions are you running?

**Comment by trenpixster** [comment]
Closed via update on #55
